backend/btrixcloud/operator.py

```diff
@@ -512,6 +515,11 @@ async def mark_finished(
         status.state = state
```

Comment:
MetaController recommends not performing long running task in the callback, and since mark_finished() might have a few of these, maybe we put all of these tasks into a separate coroutine

```
async def do_crawl_finished_tasks(cid, crawl_id):
   await update_config_crawl_stats()
   await add_crawl_errors_to_deb
   ...
   await update_crawl_collections(...)
```

and then call `asyncio.create_task(self.do_crawl_finished_tasks(...))` at the end of mark_finished()